src/usdb_syncer/notes_parser.py

```diff
@@ -252,6 +252,14 @@ def fix_pitch_values(self) -> None:
             for note in self.all_notes():
                 note.pitch = note.pitch - octave_shift * 12
 
+    def fix_apostrophes(self) -> None:
+        for note in self.all_notes():
+            note.text = (
+                note.text.replace("''", '"')  # two single quotation marks '' by "
+                .replace("`", "’")  # grave accent ` by upright apostrophe '
+                .replace("´", "’")  # acute accent ´ by upright apostrophe '
```

Comment:
This is still `’`.
To avoid duplication, I would use a function here:
```py
def replace_false_apostrophes_and_quotation_marks(value: str) -> str:
    return value.replace("''", '"')  # two single quotation marks '' by "
            .replace("`", "'")  # grave accent ` by upright apostrophe '
            .replace("´", "'"),  # acute accent ´ by upright apostrophe '
```

Reply:
Good catch! I overlooked that one. Thanks for the suggestion with the function, I added another commit.